### ISSUE TITLE
1.20.0 release

### ISSUE THREAD

**Issue by adamralph** [FakeItEasy/FakeItEasy#297]
**Ready** when all other issues forming part of the release are **Done**.
- [x] run code analysis in VS in _Release_ mode and address violations (send a regular PR which must be merged before continuing)
- [x] check build, update draft release in [GitHub UI](https://github.com/FakeItEasy/FakeItEasy/releases)
     including release notes, mentioning non-owner contributors, if any
- [x] push NuGet package
- [x] copy release notes from GitHub to NuGet
- [x] de-list pre-release NuGet packages if present
- [x] update website with contributors list (if in place)
- [x] tweet, mentioning contributors and post link as comment here for easy retweeting ;-)
- [x] post tweet in JabbR ([fakeiteasy](https://jabbr.net/#/rooms/fakeiteasy) and [general-chat](https://jabbr.net/#/rooms/general-chat)) and Gitter ([FakeItEasy/FakeItEasy](https://gitter.im/FakeItEasy/FakeItEasy))
- [x] post links to the NuGet and GitHub release in each issue in this milestone, with thanks to contributors
- [x] use `rake set_version[new_version]` to change CommonAssemblyInfo.cs to expected minor version (of form _xx.yy.zz_)
- [x] push to origin branch, create PR to upstream master
- [x] use `rake create_milestone[new_version]` to
  - create a new milestone for the next release
  - create new issue (like this one) for the next release, adding it to the new milestone
  - create a new draft GitHub Release 
- [x] close all issues on this milestone
- [x] close this milestone
